controllers/cuenta_bancaria_banco.go

```diff
@@ -46,25 +50,25 @@ func (c *CuentaBancariaBancoController) GetOne() {
 // GetAll ...
 // @Title GetAll
 // @Description get Cuenta_bancaria_banco
-// @Param	query	query	string	false	"Filter. e.g. col1:v1,col2:v2 ..."
-// @Param	fields	query	string	false	"Fields returned. e.g. col1,col2 ..."
-// @Param	sortby	query	string	false	"Sorted-by fields. e.g. col1,col2 ..."
-// @Param	order	query	string	false	"Order corresponding to each sortby field, if single value, apply to all sortby fields. e.g. desc,asc ..."
 // @Param	limit	query	string	false	"Limit the size of result set. Must be an integer"
 // @Param	offset	query	string	false	"Start position of result set. Must be an integer"
-// @Success 200 {object} models.Cuenta_bancaria_banco
-// @Failure 403
+// @Success 200 {object} []models.Cuenta_bancaria_banco
```

Comment:
`Cuenta_bancaria_banco` no existe en models, no es el tipo de dato retornado por `ObtenerCuentasBancariasBancos`

https://github.com/udistrital/tesoreria_mid/blob/1521225f4dff28290fa6502a732edf71695b1606/helpers/cuenta_bancaria_banco.go#L33

Con lo siguiente debería ser suficiente

```suggestion
// @Success 200 {object} []models.CuentaBancariaBanco
```

(Es necesario volver a regenerar los swagger.json/yml despues de lo anterior)

Reply:
Se ajusta en el commit https://github.com/udistrital/tesoreria_mid/commit/23e194b49ff6d61f80fc2c2478aa2fd791ad6cb3